### PR TITLE
[Kiwi 1357]Send Fallback Email if Placeholders Missing

### PR DESCRIPTION
**Pull request by hemavana**
### What changed

Send Fallback email if mandatory nameParts is missing required to send the static email.

### Why did it change

To make sure Users receive email even when nameParts may be missing due to event structure changes etc.

### Issue tracking

- [KIWI-1357](https://govukverify.atlassian.net/browse/KIWI-1357)

### PII logging

- [x] Verified that no PII data is being logged

[KIWI-1357]: https://govukverify.atlassian.net/browse/KIWI-1357?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: deploy/template.yaml [hunk 1]
```diff
@@ -103,7 +103,9 @@ Mappings:
     dev:
       GOVUKNOTIFYTEMPLATEID: "3baf3b59-2396-44c8-b7a4-b5a981fad5e8"
       GOVUKNOTIFYDYNAMICEMAILTEMPLATEID: "fc1a7e2f-5186-4cb8-a8e0-81110b4e9e85"
-      GOVUKNOTIFYAPI: "https://ipvr-gov-notify-stub-govnotifystub.return.dev.account.gov.uk/govnotify"
+      GOVUKNOTIFYFALLBACKMAILTEMPLATEID: "504a35f3-56f9-42f8-965c-cad58ff9b816"
+      GOVUKNOTIFYAPI: "https://api.notifications.service.gov.uk"
+      #GOVUKNOTIFYAPI: "https://ipvr-gov-notify-stub-govnotifystub.return.dev.account.gov.uk/govnotify"
```
Comment (ViveakV) [inline deploy/template.yaml:108]: Reminder to revert this back

Comment (hemavana) [reply deploy/template.yaml]: Reverted.

**Review by ViveakV — DISMISSED** [review]

File: deploy/template.yaml [hunk 2]
```diff
@@ -2099,6 +2105,42 @@ Resources:
               MetricName: IssueAccessTokenLambdaTimeOut
             Period: 60
             Stat: Sum
+  
+  StreamProcessorFunctionErorMetricFilter:
```
[the remark this answers is not in the document]
Comment (hemavana) [reply deploy/template.yaml]: Updated, hope its better now.

**Review by madankaruppiah — APPROVED** [review]
Alarm looks good to me.

**Pull request merged**